colors/vim-monokai-tasty.vim

```diff
@@ -86,7 +87,7 @@ call Highlight("Whitespace", s:dark_grey, s:none, s:none)
 
 call Highlight("TabLineSel", s:none, s:charcoal, s:bold)
 
-call Highlight("SignColumn", s:grey, s:darker_grey, s:none)
+call Highlight("SignColumn", s:none, s:none, s:none)
```

Comment:
Bit of a design preference, but I think signs like those provided by GitGutter are easier to read without the colored BG in the SignColumn.